components/automate-gateway/api/iam/v2/common/policy.proto

```diff
@@ -11,11 +11,18 @@ enum Type {
 }
 
 message Policy {
+    // Name for the policy.
     string name = 1;
+    // Unique ID for the policy. Cannot be changed after creation.
     string id = 2;
```

Comment:
Hi i'm late here but I think we should indicate that these IDs are user-specified. That might be part of @phiggins questions around ID definition, as well.

Reply:
I have `Cannot be changed after creation.` for every create request. Is it not clear that it's user specified from that? Anyway, I don't have a strong opinion about how the IDs should be annotated, but I'm not clear on what people are asking for specifically.

Reply:
IMHO the existing text is spot on because _everything_ should be assumed to be user-specified--we should only need to point out something if it is *not*. And that is just what the additional note of "Cannot be changed after creation" does.

Reply:
i think IDs are tricky because it many parts of automate we using global unique identifier.. i think auth stuff is pretty unique in regards to allowing users to specify IDs. id be interested in hearing more from the docs team on this.